### DISCUSSION TITLE
Failing right at the installation Process

### DISCUSSION THREAD

**Discussion by shibbrich** [Kozea/Radicale#1460] [Q&A general]
Hey there,

to start with: I'm a complete linux noob and I'm trying to get into this...

I did setup a Linux VM with Ubuntu and did update and upgrade and installed python3 pip. 

I Read the instructions (although i didnt fully understand everything) and tried to install Radicale.
First command did work just fine.
`python3 -m pip install --upgrade https://github.com/Kozea/Radicale/archive/master.tar.gz`
Second command didnt work at all:
`python3 -m radicale --storage-filesystem-folder=~/.var/lib/radicale/collections`

Executing the second one does nothing for me. I execute the command and it just jumps down one line and the marker flashes... i can type what ever i want and nothing happens.
![radicale](https://github.com/Kozea/Radicale/assets/141817679/baeed835-cfe8-45a9-9d32-71af8d96eea8)

Can someone help me here?

**Comment by shibbrich** [comment]
Even Trying the v3 doenst work. Same issue.
What am i missing?

![image](https://github.com/Kozea/Radicale/assets/141817679/3b227c92-1c67-4824-963b-d0f4ddc0d767)

**Comment by pbiering** [comment]
the instructions miss some update regarding the dropped daemon mode in 3.x, please try the "simple" part from now updated Wiki:
https://github.com/Kozea/Radicale/wiki/Simple-installation

Will update the main doc next

**Comment by pbiering** [comment]
Changed default loglevel to "info" by https://github.com/Kozea/Radicale/pull/1462 to make bootstrapping easier ;-)
